frontend/src/v5/ui/components/dashboard/dashboardViewerLayout/dashboardViewerLayout.component.tsx

```diff
@@ -17,14 +17,33 @@
 import { ThemeProvider } from 'styled-components';
 import { ThemeProvider as MuiThemeProvider } from '@mui/material';
 import { theme } from '@/v5/ui/routes/viewer/theme';
-import { DashboardLayout } from '../dashboardLayout/dashboardLayout.component';
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
 
-export const DashboardViewerLayout = ({ children }) => (
-	<DashboardLayout>
-		<ThemeProvider theme={theme}>
-			<MuiThemeProvider theme={theme}>
-				{children}
-			</MuiThemeProvider>
-		</ThemeProvider>
-	</DashboardLayout>
-);
+import { AppBar } from '@components/shared/appBar';
+import { TeamspacesActionsDispatchers, ProjectsActionsDispatchers } from '@/v5/services/actionsDispatchers';
+import { DashboardParams } from '@/v5/ui/routes/routes.constants';
+import { Content } from './dashboardViewerLayout.styles';
+
+export const DashboardViewerLayout = ({ children }) => {
+	const { teamspace, project } = useParams<DashboardParams>();
+
+	useEffect(() => {
+		ProjectsActionsDispatchers.fetch(teamspace);
```

Comment:
This isn't a change request I'm just wondering where these dispatchers have come from as I can't see them deleted in other files (?). Did the change in layout cause bugs with the redux state?

Reply:
DashboardLayout has been renamed to "DashboardProjectLayout", making it more specific, and the viewer layout does not use it anymore inside it's JSX, but it still needs to have a current project and a current teamspace being set.
As such, that logic (which was only in DashboardLayout) has been added here as well